From fb6b020b244428039fa9bba7f40b764d947d5931 Mon Sep 17 00:00:00 2001
From: John Lees <lees.john6@gmail.com>
Date: Wed, 11 Mar 2020 19:15:06 +0000
Subject: [PATCH] Use conda gfortran

---
 setup.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index cc8855a..32ba472 100644
--- a/setup.py
+++ b/setup.py
@@ -13,7 +13,7 @@ class GfortranExtension(Extension):
 class GfortranBuild(build_ext):
     def run(self):
         try:
-            out = subprocess.check_output(['gfortran', '--version'])
+            out = subprocess.check_output('${FC:-gfortran} --version', shell=True)
         except OSError:
             raise RuntimeError("gfortran must be installed to build the following extensions: " +
                                ", ".join(e.name for e in self.extensions))
@@ -32,7 +32,8 @@ class GfortranBuild(build_ext):
             os.makedirs(self.build_temp)
 
         env = os.environ.copy()
-        subprocess.check_call(['gfortran', ext.input] + gfortran_args, cwd=self.build_temp, env=env)
+        subprocess.check_call('${FC:-gfortran} ' + " ".join([ext.input] + gfortran_args),
+                              cwd=self.build_temp, env=env, shell=True)
 
         extdir = os.path.join(os.path.abspath(os.path.dirname(self.get_ext_fullpath(ext.name))),
                                               os.path.basename(ext.sourcedir))
-- 
2.25.0

